src/main/java/create_task_use_case/CreateTaskOutputData.java

```diff
@@ -0,0 +1,47 @@
+package create_task_use_case;
+
+/**
+ * Task Creation Output Data Class. Instances of this class are returned upon the
+ * execution of the application logic for this use case. The errorMessage and
+ * isSuccessfullyModified instance attributes, in particular, represent whether the use
+ * case logic was successfully executed.
+ * @author Fardin Faruk
+ */
+public class CreateTaskOutputData {
+
+    private final String title;
+    protected boolean isSuccessfullyCreated;
```

Comment:
Why is this boolean protected instead of private?

Reply:
cant be accessed by test cases otherwise

Reply:
fixed.